action.yml

```
@@ -75,13 +81,15 @@ runs:
       env:
         _ACTION_SESSIONS: ${{ inputs.sessions }}
         _ACTION_FORCE_PYTHONS: ${{ inputs.force-pythons }}
+        _SESSION_ARGS: ${{ inputs.extra-args }}
```

```suggestion
        _ACTION_EXTRA_ARGS: ${{ inputs.extra-args }}
```

would probably be more consistent with the other variable names.

---

Applied